src/index.html

```diff
@@ -9,6 +9,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <div class="containerLogo">
+        <a href="#home" class="linkLogo">
+          <img
+            class="logo"
+            src="../src/images/logo.png"
+            alt="Store logo MOYO"
+          >
+        </a>
+      </div>
+
+      <nav class="navBlock">
+
+        <ul class="ulLink">
+          <li class="navList">
+            <a href="#" class="is-active link">Apple</a>
+          </li>
+          <li class="navList">
+            <a href="#" class="link">Samsung</a>
+          </li>
+          <li class="navList">
+            <a href="#" class="link">Smartphones</a>
+          </li>
```

Comment:
```suggestion
          <li class="navList">
            <a href="#" class="is-active link">Apple</a>
          </li>
          
          <li class="navList">
            <a href="#" class="link">Samsung</a>
          </li>
          
          <li class="navList">
            <a href="#" class="link">Smartphones</a>
          </li>
```
Please fix all cases